src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -112,9 +112,10 @@ export class DictionaryModal extends React.Component {
        supported_locales,
      },
    } = this.props;
-   if (supported_locales === undefined) {
+   if (supported_locales === undefined || supported_locales === null) {
      return null;
    }
+   console.log("SUPPORTED LOCALES ", supported_locales);
```

Comment:
@emmabaye I don't think this console.log is relevant.

Reply:
Thanks for the feedback, @alexmochu. I have removed that line of code.